test/fs.test.ts

```diff
@@ -0,0 +1,65 @@
+import 'mocha';
+import * as path from 'path';
+import { expect } from 'chai';
+
+import * as fs from '../src/util/fs';
+
+describe('UTIL: fs', () => {
+
```

Comment:
Hint: you can create a physical file (temporary) under our temp directory and cleanup for each of these tests - you can use before / after hooks for that.

cc @mesaugat @SafalPandey

Reply:
FYI: https://nodejs.org/api/fs.html#fs_fs_mkdtemp_prefix_options_callback